python/lsst/daf/butler/registry/obscore/_records.py

```diff
@@ -188,6 +189,9 @@ def __call__(self, ref: DatasetRef, context: SqlQueryContext) -> Record | None:
             if timespan.end is not None:
                 t_max = cast(astropy.time.Time, timespan.end)
                 record["t_max"] = t_max.mjd
+        else:
+            record["t_min"] = -math.inf
+            record["t_max"] = math.inf
```

Comment:
I don't think that SQL in general supports +/-infinity. It might work with the backends that we support, but we probably should add a unit test for it to be sure